### PR TITLE
Overriding high voltage routes

### PR DESCRIPTION
**Pull request by JensDebergh**

### CODE REVIEW

File: spec/controllers/pages_controller_spec.rb [hunk 1]
```diff
@@ -0,0 +1,14 @@
+require "rails_helper"
+
+describe PagesController, "#show" do
+  %w(home).each do |page|
+    context "on GET to /pages/#{page}" do
```
Comment (FalkoJoseph) [inline spec/controllers/pages_controller_spec.rb]: Should we serve static pages under `/pages` though? Why not directly `/signup` `/login`. And the app pages under something like `/podcasts`, `/profile` etc. :D or how exactly do you see this?

Maybe we should set up some base routes in this PR? :)

Comment (JensDebergh) [inline spec/controllers/pages_controller_spec.rb]: We should have routes like that, but I do not yet know how to do so after overriding the defaults of `high_voltage`. I'll have a look at it.

Comment (FalkoJoseph) [inline spec/controllers/pages_controller_spec.rb]: For future reference: http://geekmonkey.org/articles/9-static-pages-with-high-voltage

Comment (JensDebergh) [inline spec/controllers/pages_controller_spec.rb]: Found it.
Just had to apply :constraints => HighVoltage::Constraints::RootRoute
So it looks for corresponding view, if found renders it. If not continues to search the route tree.

**Pull request merged**